src/main/java/edu/cornell/kfs/module/purap/document/authorization/IWantDocumentPresentationController.java

```diff
@@ -146,8 +194,32 @@ public Set<String> getEditModes(Document document) {
         
         editModes.add(CUPurapConstants.I_WANT_DOC_EDIT_PROC_NET_ID);
         editModes.add(CUPurapConstants.IWNT_DOC_DISPLAY_NOTE_OPTIONS);
+        
+        if(isContractFunctionalityEnabled()) {
```

Comment:
Another line missing whitespace after the "if" keyword.

Reply:
updated